LiskHQ/lisk-desktop
# Implement desktop notifications for incoming transactions - Closes #2460

### What issue have I solved?

Resolves #2460

### How have I implemented/fixed it?

Based on Electron docs https://electronjs.org/docs/tutorial/notifications

- [x] Ensure notifications work on MacOS
- [x] Ensure notifications work on Linux 
- [ ] Ensure notifications work on Windows


### How has this been tested?

- Open the app in Electron wrapper
- Log in to an account
- Send an incoming transaction to that account
- See the notification
  - Linux
  ![Screenshot 2020-01-03 at 15 46 50](https://user-images.githubusercontent.com/1254342/71803024-ea59c700-305f-11ea-8624-9ee0a18a429f.png)
  - MacOS
  ![Screenshot 2020-01-03 at 15 46 50](https://user-images.githubusercontent.com/1254342/71804077-bb912000-3062-11ea-8225-b5b20643c6ec.png)

  - Windows TODO

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)
